hyperspace/core/src/command.rs

```diff
@@ -111,7 +119,48 @@ impl Cmd {
 			any_chain_b.name(),
 			client_id_a_on_b
 		);
-		Ok(())
+
+		let parachain_client_a = ParachainClient::try_from(any_chain_a).unwrap();
+		let config_parachain_a = ParachainClientConfig::try_from(config.chain_a).unwrap();
+		let new_parachain_config_a = ParachainClientConfig {
+			name: parachain_client_a.name,
+			para_id: parachain_client_a.para_id,
+			parachain_rpc_url: config_parachain_a.parachain_rpc_url,
+			relay_chain_rpc_url: config_parachain_a.relay_chain_rpc_url,
+			client_id: parachain_client_a.client_id,
+			connection_id: parachain_client_a.connection_id,
+			beefy_activation_block: parachain_client_a.beefy_activation_block,
+			commitment_prefix: parachain_client_a.commitment_prefix.into(),
+			private_key: config_parachain_a.private_key,
+			ss58_version: config_parachain_a.ss58_version,
+			channel_whitelist: parachain_client_a.channel_whitelist,
+			finality_protocol: parachain_client_a.finality_protocol,
+			key_type: config_parachain_a.key_type,
+		};
+
+		let parachain_client_b = ParachainClient::try_from(any_chain_b).unwrap();
+		let config_parachain_b = ParachainClientConfig::try_from(config.chain_b).unwrap();
+		let new_parachain_config_b = ParachainClientConfig {
+			name: parachain_client_b.name,
+			para_id: parachain_client_b.para_id,
+			parachain_rpc_url: config_parachain_b.parachain_rpc_url,
+			relay_chain_rpc_url: config_parachain_b.relay_chain_rpc_url,
+			client_id: parachain_client_b.client_id,
+			connection_id: parachain_client_b.connection_id,
+			beefy_activation_block: parachain_client_b.beefy_activation_block,
+			commitment_prefix: parachain_client_b.commitment_prefix.into(),
+			private_key: config_parachain_b.private_key,
+			ss58_version: config_parachain_b.ss58_version,
+			channel_whitelist: parachain_client_b.channel_whitelist,
+			finality_protocol: parachain_client_b.finality_protocol,
+			key_type: config_parachain_b.key_type,
+		};
+		let new_config = Config {
+			chain_a: AnyConfig::Parachain(new_parachain_config_a),
+			chain_b: AnyConfig::Parachain(new_parachain_config_b),
+			core: config.core,
+		};
+		Ok(new_config)
 	}
 
 	pub async fn create_connection(&self) -> Result<()> {
```

Comment:
We should use the same techniques for connections and channels.
`set_connection_id`, `set_channel_whitelist`